jpeg_benchmark.py

```diff
@@ -0,0 +1,175 @@
+import gc
+import io
+import json
+import os
+import pathlib
+import time
+import datetime
+from PIL import Image
+
+# Setting parameters
+FRAME_COUNT = 300
+REAL_FPS = 30
+FRAME_TO_SAVE = 69
+INPUT_PATH = pathlib.Path("test_images", "Encode Test Dataset 2024")
+OUTPUT_PATH = pathlib.Path(f"log_{int(time.time())}")
+ENCODING = "jpeg"
+NUM_REPEATS = 1
+
+# Keys for dictionary entries
+TOTAL_TIME_MS = "total_time_ms"
+AVG_TIME_MS = "avg_time_ms"
+TOTAL_SPACE_B = "total_space_B"
+AVG_SPACE_B = "avg_space_B"
+FRAME_SPACE = "frame_space"
+FRAME_DATA = "frame_data"
+
+STIMULATED_FPS = [1, 5, 10, 30] # Test image compression, FPS
```

Comment:
Oh, sorry for the confusion. We no longer need this since we are doing image transmission only. Look at the main branch for references. (the old branches are outdated)